packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +44,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.title) {
```

Comment:
@kenk2 - thanks for the suggestion:
```
frameOptions.iframeAttrs = this.props.title && frameOptions.iframeAttrs ? frameOptions.iframeAttrs : {}
```
I'll rework the logic a bit.

Reply:
I re-worked the logic a bit in https://github.com/cerner/terra-framework/pull/1776/commits/3ffd8305ee593960880ba6b3ac9bbdab84932e6c